ext/civigrant/tests/phpunit/api/v3/GrantTest.php

```diff
@@ -133,14 +151,27 @@ public function testCreateCustomCheckboxGrant() {
     $this->assertEquals($expectedValue, $savedValue);
   }
 
-  public function testGetGrant() {
-    $result = $this->callAPISuccess($this->_entity, 'create', $this->params);
+  public function testGetGrant(): void {
+    $result = $this->callAPISuccess('Grant', 'create', $this->params);
     $this->ids['grant'][0] = $result['id'];
     $result = $this->callAPISuccess($this->_entity, 'get', ['rationale' => 'Just Because']);
     $this->assertGreaterThanOrEqual(1, $result['count']);
   }
 
-  public function testDeleteGrant() {
+  public function testGrantGetDoesNotInterfereWithContributionGet() {
```

Comment:
@eileenmcnaughton this not returning void? :)

Reply:
yeah - @seamuslee001 odd that my IDE doesn't care about that one - pushed it up tho